backend/src/services/mail_verification/send.service.ts

```diff
@@ -0,0 +1,42 @@
+import { ENV } from '@common/enums/app/app';
+import nodemailer from 'nodemailer';
+
+import { generateTokens } from './token.service';
+
+const transporter = nodemailer.createTransport({
+  host: ENV.MAIL.SMTP_HOST,
+  port: ENV.MAIL.SMTP_PORT,
+  secure: false,
+  auth: {
+    user: ENV.MAIL.SMTP_USER,
+    pass: ENV.MAIL.SMTP_PASSWORD,
+  },
+});
+
+async function sendActivationMail(to: string, link: string): Promise<void> {
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/17#discussion_r943544259

Reply:
I left it to Serhii, who does this task.